app/controllers/home_controller.rb

```diff
@@ -161,7 +160,11 @@ def activity
   end
 
   def trending
+    # Trending tags exculding subscriptions and tag first time poster tag
+    # The first time poster tag is not useful in search
     exclude_tids = @tag_subscriptions.pluck(:tid)
-    Tag.trending.where.not(tid: exclude_tids)
+    exclude_tids << Tag.where(name: %w(everything first-time-poster)).first&.tid
```

Comment:
This query looks fishy...I'll fix it.

Reply:
Ok! Thanks! Ping me when it's ready to merge! 🎉

Reply:
@jywarren, The tests are now passing and the issue above has been fixed. You can merge once you've reviewed the changes I have made, thanks!